src/main/webapp/styles/style.css

```diff
@@ -261,3 +260,28 @@ body {
     transition: font-size 0.3s ease-in-out;
   }
 }
+
+.loader {
+  animation: spin 2s linear infinite;
+  border: 16px solid #f3f3f3;
+  border-bottom: 10px solid yellow;
+  border-left: 10px solid green;
+  border-radius: 50%;
+  border-right: 10px solid red;
+  border-top: 10px solid blue;
+  height: 50px;
+  margin-left: 420px;
+  margin-top: 20px;
+  -webkit-animation: spin 2s linear infinite;
+  width: 50px;
+}
+
+@-webkit-keyframes spin {
```

Comment:
I'm curious to see this in action (it's beyond my CSS knowledge which is a few years behind).

Reply:
Sure! I will deploy so you can see it as well.